src/locales/hu.yaml

```diff
@@ -1,508 +1,499 @@
-      date_time_format: Dátum és idő formátum
-    timer_options:
-      duration: Csak időtartam
-      filament: Nyomtatószál becslés
-      file: Fájlbecslés
-      slicer: Szeletelő
-    title:
-      authentication: Hitelesítés
-      console: Konzol
-      camera: Kamera
-      file_editor: Fájlszerkesztő
-      gcode_preview: Gcode előnézet
-      general: Általános
-      macros: Makrók
-      theme: Téma
-      thermal_presets: Hőmérsékleti előbeállítások
-      tool: Eszköz
-    tooltip:
-      gcode_coords: >-
-        Használja a GCode pozíciót a szerszámfej pozíciója helyett az
-        irányítópulton
-  socket:
-    msg:
-      connecting: Csatlakozás a moonrakerhez...
-      no_connection: >-
-        Nincs moonraker kapcsolat. Kérjük, ellenőrizd a moonraker állapotát
-        és/vagy frissítsd.
-  system_info:
-    label:
-      capacity: Kapacitás
-      cpu_desc: CPU leírása
-      distribution_codename: Kód név
-      distribution_like: Alap disztribúció
-      distribution_name: Disztribúció
-      hardware_desc: Hardver leírás
-      hostname: Gazdanév
-      klipper_load: Klipper Terheltsége
-      manufactured: Gyártva
-      manufacturer: Gyártó
-      mcu_awake: '{mcu} Ébredés ideje'
-      mcu_bandwidth: '{mcu} Sávszélesség'
-      mcu_load: '{mcu} Terheltsége'
-      model: CPU Modell
-      moonraker_load: Moonraker Terheltsége
-      processor_desc: Processzor
-      product_name: Termék név
-      serial_number: Sorozatszám
-      system_load: Rendszer Terheltsége
-      system_memory: Rendszermemória
-      system_utilization: Rendszerhasználat
-      total_memory: Teljes memória
-  tool:
-    btn:
-      home_x: X
-      home_y: 'Y'
-    tooltip:
-      extruder_disabled: extruder letiltva, min_extrude_temp (%{min}<small>°C</small>) alatt van
-      home_xy: XY kezdőpont
-      home_z: Z kezdőpont
-  version:
-    btn:
-      check_for_updates: Frissítések keresése
-      finish: Befejezve
-      update: Frissítés
-      view_versions: Verziók megtekintése
-    label:
-      commit_history: Történet rögzítése
-      commits_on: vállalja
-      committed: Elkötelezett
-      dirty: PISZKOS
-      invalid: ÉRVÉNYTELEN
-      os_packages: OS csomagok
-      package_list: Csomaglista
-      up_to_date: NAPRAKÉSZ
-      updates_available: Frissítések állnak rendelkezésre
-    status:
-      finished: A frissítések befejeződtek
-      updating: Frissítés...
-    title: Szoftverfrissítések
-    tooltip:
-      commit_history: Előzmények rögzítése
-      dirty: levált fejet jelez, nem a mesteren vagy érvénytelen eredetet
-      invalid: a repo helyi változásait jelzi
-      packages: Csomagok
-      release_notes: Kiadási megjegyzések
+app:
+  bedmesh:
+    label:
+      box_scale: Skála
+      flat_surface: Lapos sík megjelenítése
+      mesh_matrix: Hálós mátrix
+      probed_matrix: Próbamátrix
+      profile_name: Profil Név
+      remove_profile: %{name} profil eltávolítása

```

Comment:
The single quote `'` around this translation needs to be kept as this starts with a special replacement sequence.